### PR TITLE
Second Milestone: Add sections to Mobile website portfolio

### PR DESCRIPTION
**Pull request by Brenda309**
### In this pull request, I:

- [x] Add project section of my portfolio.
- [x] Add about the self section of my portfolio

### CODE REVIEW

**Review by codepantherr — CHANGES_REQUESTED** [review]
## Hi @Brenda309 👋, 

What an awesome project you have here! You should be proud.. 🦾 
There are some issues that you still need to work on to go to the next project but you are almost there!


## To Highlight ⚡ 
- Meaningful PR ✔️ 
- Detailed commits ✔️ 
- Beautiful UI ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes ♻️

- [x] Please make your README a little bit more descriptive especially the **Getting Started** section. You typically want to guide users through the steps to set clone your repo and set up the project locally. 👍 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -37,6 +38,204 @@ <h2><a href="#"> Let's connect</a></h2>
             <li><a href="#" class="peace"><img src="./images/peace.svg" alt="peace"></a></li>
         </ul>
     </section>
+
+    <div class="projects-container">
+        <section class="portfolio project-card">
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply index.html]: Well implemented 👍🏽

File: style.css [hunk 2]
```diff
@@ -74,3 +79,177 @@ h2 {
...
+}
+
+.clickme {
+  color: #396df2;
+  border-radius: 8px;
+  border-color: hsl(223, 88%, 59%);
+  padding: 10px 20px;
+  background-color: #fff;
+  font-size: 20px;
+}
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply style.css]: - [ ] [REQUIRED] You have only implemented the `:hover` pseudo-class but not `:pressed`. Kindly go ahead to implement this has it is a requirement.

File: index.html [hunk 3]
```diff
@@ -37,6 +38,204 @@ <h2><a href="#"> Let's connect</a></h2>
...
+        <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+        </ul>
+        <div><button class="clickme" type="button">See Project</button></div>
+    </section>
+
+    <section class="about-me">
+        <h3 class="self">About <br> myself</h3>
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply index.html]: - [ ] [REQUIRED] Just as the previous reviewer recommended. You have not stuck to the color and fonts as provided in the design template. There ought to be a grey background then a white background with a border for the cards and the border for the about myself container See below an explanatory image:

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot from 2022-02-19 17-37-41 | Screenshot from 2022-02-19 17-37-35

File: index.html [hunk 4]
```diff
@@ -37,6 +38,204 @@ <h2><a href="#"> Let's connect</a></h2>
...
+            <li><a href="#" class="linkedin"><img src="./images/linkdin.svg" alt="linkdin"></a></li>
+            <li><a href="#" class="Mediam"><img src="./images/median.svg" alt="Mediam"></a></li>
+            <li><a href="#" class="GitHub"><img src="./images/github.svg" alt="GitHub"></a></li>
+            <li><a href="#" class="peace"><img src="./images/peace.svg" alt="peace"></a></li>
+        </ul>
+        <div><button class="clickme" type="button">Get My Resume</button></div>
+        <div class="about-languages">
+            <div class="stacks">
+                <div class="heading">
+                    <h4> Languages </h4>
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply index.html]: Well implemented 👍🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

- [ ] [REQUIRED] As recommended by the last reviewer, kindly modify your README file. You actually added the steps to get it cloned to a machine locally but you have to put the URL of this project not just `github.com`. Please fix this.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (Brenda309) [reply style.css]: Hello @Whoistolu the pseudo-class but not :pressed I have looked almost everywhere and I can't find anything to refer to it. and I also tried to just type implementing it in my css but the linters are throwing errors saying that it doesn't exist. Also if you may consider looking at the requirement document "https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md" check on the last line,  only says the pseudo-class hover and enable mentioned there isn't such a thing like :pressed. I can not forget to thank you for the helpful feedback you gave.

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @Brenda309 ,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Linters are passing :heavy_check_mark: 

**Very good job implementing the changes asked by previous reviewers :clap:**
What happens is that my fellow peers wanted you to search and discover by yourself the pseudo-classes that you can use in the project. Here is a link that I highly recommend for you to study: [CSS pseudo classes](https://www.webfx.com/blog/web-design/link-pseudo-classes/).
I recommend as well as previous reviewers, to implement the state of a pressed button, with the help of the pseudo-class of `:visited`.
Here is what a visited/pressed button should look like:
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**